src/components/MainLayout/MainHeader.tsx

```diff
@@ -48,53 +51,60 @@ export const MainHeader = () => {
     <Box
       css={{
         px: '$1xl',
-        pt: '9px', // manual offset to align breadcrumb
-        height: '80px',
+        height: '91px',
         display: 'flex',
-        justifyContent: 'space-between',
-        alignItems: 'center',
-        background: '$text',
+        flexDirection: 'column',
+        background: '$headingText',
       }}
     >
-      <Image
-        alt="logo"
-        css={{ height: '$1xl', mr: '$md' }}
-        src="/svgs/logo/logo.svg"
-      />
-      {hasCircles && <TopLevelLinks links={mainLinks} />}
-
+      <Box
+        css={{
+          mt: 'calc($sm - 1px)',
+          mb: '-2px',
+          pb: '$xs',
+          px: '20px',
```

Comment:
Should use a theme token instead of a hard coded value here

Reply:
Yea, this one is a pretty tricky one to position exactly.  There weren't any tokens that got the alignment right (this is for the COORDINAPE brand text above the OverviewMenu button.  I'll see what I can do to token-ize it

Reply:
I would put the onus on the designers to update the figma so it conforms to the standard sizing / spacing tokens.

In the designs there is much less padding on the top level box, so you could remove the `px: '$1xl'` there on line 53 and then just change this to `px: '$lg'` which matches the 24px in the designs.

The designs incorrectly have a 12px space between the overview menu button and the left edge, the space scale dictates it should be either 8px or 16px.

Don't have to deal with it in this PR if you dont want to, but would atleast put a TODO comment here and let design team know

![image](https://user-images.githubusercontent.com/7143583/170185308-e9f99d42-5b18-4ddf-aba8-2bc650388639.png)